### ISSUE TITLE
build form local dockerfile?

### ISSUE THREAD

**Issue by v0nNemizez** [fussybeaver/bollard#290]
i might need glasses, but i cant see anywhere in the documentation that indicate that this is pausible?
I have a git repo where i have my custom config and Dockerfiles that i pull on a regular basis when i change configs and so on.

**Comment by fussybeaver** [comment]
No, this isn't implemented (yet). In theory, this would be possible using a 'client-session' remote context and the v2 buildkit builder, but... there's some further protobuf communication and serialization interfaces to add, so it's non-trivial.

**Comment by v0nNemizez** [comment]
okay :) no eyesight problems then! i was worried xD  Thx for the response :) you might consider this a Futrure Request then :)
